mindjunkies/dashboard/tests/test_admin.py

```
@@ -0,0 +1,118 @@
+        assert certificate_admin.search_fields == ("description",)
+        
+    def test_list_filter(self, admin_site):
+        certificate_admin = CertificateAdmin(Certificate, admin_site)
+        assert certificate_admin.list_filter == ("created_at", "updated_at")
+        
+    def test_certificate_creation(self, admin_site):
+        # Create a certificate using model_bakery
+        certificate = baker.make(Certificate, description="Test Certificate")
+        
+        # Set up the admin
+        certificate_admin = CertificateAdmin(Certificate, admin_site)
```

Ruff (0.8.2)

Local variable `certificate_admin` is assigned to but never used

Remove assignment to unused variable `certificate_admin`

(F841)